test/ScriptCs.Engine.Roslyn.Tests/CSharpScriptInMemoryEngineTests.cs

```diff
@@ -43,12 +43,12 @@ public void ShouldExposeExceptionThrownByScriptWhenErrorOccurs()
             [Fact]
             public void ShouldExposeExceptionThrownByCompilation()
             {
-                var scriptEngine = new RoslynScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
+                var scriptEngine = new CSharpScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
 
                 // Arrange
                 var lines = new List<string>
                 {
-                    "using Sysasdasdasdtem;"
+                    "Sysasdasdasdtem;"
```

Comment:
I don't believe it supports Mono currently.

On Thu, Aug 13, 2015 at 12:00 PM Adam Ralph <notifications@github.com>
wrote:

> In test/ScriptCs.Engine.Roslyn.Tests/CSharpScriptInMemoryEngineTests.cs
> <https://github.com/scriptcs/scriptcs/pull/1064#discussion_r37012264>:
>
> >
> >                  // Arrange
> >                  var lines = new List<string>
> >                  {
> > -                    "using Sysasdasdasdtem;"
> > +                    "Sysasdasdasdtem;"
>
> Why did this have to change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/1064/files#r37012264>.
>